testutil-gcloud/src/test/java/io/spine/testing/server/storage/datastore/TestDatastoresTest.java

```diff
@@ -116,7 +118,9 @@ void byResource() {
     @Test
     @DisplayName("throw an `ISE` when can't properly parse account credentials from resource")
     void throwOnInvalidResource() {
-        Resource resource = Resource.file("random.json");
+        Resource resource = Resource.file(
```

Comment:
Can we have a shortcut for both usages of this `Resource.file` in this class? 

It's just that they both don't fit a single line. And it would be nice to write this code just once.